src/test/resources/val-var-def-one-liners/expected.txt

```diff
@@ -61,11 +61,25 @@ Relevant bindings include
   val hole13: (Int, String) => (String, Int) = { (a, b) => ??? }
                                                            ^
 src/test/resources/val-var-def-one-liners/input.scala:31:
-Found hole with type: Int
+Found hole with type: B
```

Comment:
I feel like if I can find the right incantation I can persuade the compiler to fill in this type param with the concrete type. More experimentation needed...